string in_
---
std_msgs/Header header
std_msgs/Bool done
std_msgs/Bool crash
sensor_msgs/Image image
geometry_msgs/Point current_position
geometry_msgs/Point goal_position